manifests/server/rack.pp

```diff
@@ -16,9 +16,13 @@
   $rack_arguments = $::puppet::server_rack_arguments,
   $user           = $::puppet::server_user,
   $vardir         = $::puppet::vardir,
+  $touch_cmd      = $::osfamily ? {
+    /^(FreeBSD|DragonFly)$/ => '/usr/bin/touch',
+    default                 => '/bin/touch'
```

Comment:
Not 100% sure about that, as I read that historically only RedHat/Fedora had this in /bin. However, Debian has a symlink by default, so this never got noticed.